dead.py

```diff
@@ -55,12 +57,19 @@ def visit_ImportFrom(self, node: ast.ImportFrom) -> None:
 
     def visit_ClassDef(self, node: ast.ClassDef) -> None:
         if not self.is_test:
-            self.defines[node.name].add(self.filename)
+            self.mark_definition(node)
         self.generic_visit(node)
 
+    def mark_definition(self, node: DefinitionAST) -> None:
+        """Marks the code defined by `node` as defined"""
```

Comment:
I think the function name is clear enough that the docstring doesn't add anything that isn't already clear